app/src/main/java/org/sopt/and/screen/SignUpScreen.kt

```diff
@@ -0,0 +1,257 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.R
+import org.sopt.and.component.SignUpTextField
+
+val passwordRegex =
+    "^(?=.*[A-Z])(?=.*[a-z])(?=.*\\d)(?=.*[@#\$%^&+=!])(?=\\S+$).{8,20}$".toRegex()
+
+@Composable
+fun SignUpScreen(onRegister: (String, String) -> Unit, modifier: Modifier = Modifier) {
+    var showPassword by remember { mutableStateOf(false) }
+    var id by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    val context = LocalContext.current
+
+    fun dataCheck() {
+        id = id.trim()
+        password = password.trim()
+        if (!Patterns.EMAIL_ADDRESS.matcher(id).matches()) {
+            Toast.makeText(context, "이메일 형식에 맞지 않습니다.", Toast.LENGTH_SHORT).show()
+        } else if (!passwordRegex.matches(password)) {
+            Toast.makeText(
+                context, " 비밀번호는 8~20자 이내 영문 대소문 , 숫자, 특수문자 중\" +\n" +
+                        "            \"\n 3가지 이상 혼용하여 사용해주세요.", Toast.LENGTH_SHORT
+            ).show()
+        } else {
+            onRegister(id, password)
+            Toast.makeText(context, "회원가입에 성공하였습니다", Toast.LENGTH_SHORT).show()
+        }
+    }
+
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = colorResource(R.color.basic_background))
+            .padding(10.dp)
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier.fillMaxWidth()
+        ) {
+            Spacer(Modifier)
+            Text(
+                text = stringResource(R.string.signup),
+                fontSize = 20.sp, modifier = Modifier, color = colorResource(R.color.white)
+            )
+            Icon(
+                imageVector = Icons.Default.Clear, contentDescription = Icons.Default.Clear.name,
+                modifier = Modifier
+                    .size(30.dp)
+                    .clickable {
+                        (context as Activity).finish()
+                    },
+                tint = colorResource(R.color.exit)
+            )
+        }
+
+        Spacer(Modifier.padding(13.dp))
+
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalAlignment = Alignment.Start
+        ) {
+            Text(
+                text = "이메일과 비밀번호만으로" +
+                        "\n" + "Wavve를 즐길 수 있어요!",
```

Comment:
```suggestion
                text = "이메일과 비밀번호만으로\nWavve를 즐길 수 있어요!",
```
요렇게 한 번에 적으실 수도 있을 것 같아요!